src/cmds/qdel.c

```diff
@@ -241,25 +327,23 @@ char **envp;
 			pbs_strncpy(warg, warg1, sizeof(warg));
 		}
 
-		stat = pbs_deljob(connect, job_id_out, warg);
+		p_delstatus = pbs_deljoblist(connect, pbs_svr->jobids, warg);
+		
+		while (p_delstatus != NULL) {
+			if ((pbse_to_txt(p_delstatus->code) != NULL) && (p_delstatus->code != PBSE_HISTJOBDELETED)) {
```

Comment:
store output of `pbse_to_txt()` in some variable and use in below like instead calling it again

Reply:
okay, I agree, I will change it.